packages/contracts/contracts/core/permission/PermissionManager.sol

```diff
@@ -181,8 +181,8 @@ contract PermissionManager is Initializable {
             BulkPermissionsLib.ItemMultiTarget memory item = items[i];
 
             // TODO: Optimize
-            _auth(item.where, item.permissionId);
-            
+            _auth(item.where, ROOT_PERMISSION_ID);
```

Comment:
Why root permission here?

Reply:
I agree with @brickpop, this doesn't seem necessary, but needs very thorough review. 
I think this is a mix-up. Maybe @novaknole wanted to apply this to `function bulkOnMultiTarget` instead of `function bulkOnSingleTarget`, which is also the goal of https://github.com/aragon/core/pull/90. 
Let's fix the problem in permission manager in the separate PR.

Reply:
```suggestion
            _auth(item.where, item.permissionId);
```

Reply:
is issue is addressed in this PR: https://github.com/aragon/core/pull/90